### DISCUSSION TITLE
sx vs createStyles

### DISCUSSION THREAD

**Discussion by karlas** [mantinedev/mantine#3615] [Q&A] [answered]
Hi,

When it comes tu create parametrized styles, I use to implement a function accepting these parameters which returns a `sx` function.

Please checkout https://codesandbox.io/s/keen-lena-fzisr7?file=/src/App.tsx

However, at https://mantine.dev/styles/create-styles/ it says using `createStyles` is the recommended way to do it.

Is there any **performance** advantage using `createStyles` over my approach?

Thanks in advance

**Comment by rtivital** [comment, marked as answer]
No, there is no difference, sx and createStyles work the same way
